### PR TITLE
chore(test): update flattened schema fixture to match actual schema

### PR DESCRIPTION
**Pull request by boneskull**
This fixes the test that is making CI fail.

### CODE REVIEW

**Comment by boneskull** [comment]
It'd be nice if this fixture got auto-updated when the schema changes, but the only way to do that is to use the code that's under test, which means it'd defeat the point of the test.

File: packages/appium/lib/schema/schema.js [hunk 1]
```diff
@@ -576,17 +579,19 @@ export class SchemaNameConflictError extends Error {
...
    * @param {string} extName
+   * @param {{conflictedSchema?: SchemaObject}} [opts]
    */
-  constructor(extType, extName) {
-    super(`Name for ${extType} schema "${extName}" conflicts with an existing schema`);
-    this.data = {extType, extName};
+  constructor(extType, extName, {conflictedSchema} = {}) {
+    super(`Name for ${extType} schema "${extName}" conflicts with an existing schema.`);
+    this.data = {extType, extName, conflictedSchema};
+    logger.debug(JSON.stringify(this.data, null, 2));
```
Comment (jlipps) [inline packages/appium/lib/schema/schema.js]: we want to print something just when the error is created? is this kind of an equivalent to our  old pattern of using `errAndThrow`?

File: packages/appium/test/fixtures/flattened-schema.js [hunk 2]
```diff
@@ -522,7 +522,7 @@ export default [
     },
     schema: {
       appiumCliDescription:
-        'A list of drivers to activate. Can be a comma-delimited string or path to CSV file. By default, all installed drivers will be activated.',
+        'A list of drivers to activate. Can be a comma-delimited string or path to CSV file. By default, all installed drivers will be activated. Windows environments may require wrapping the comma-delimited string with quotes to escape the comma.',
```
Comment (jlipps) [inline packages/appium/test/fixtures/flattened-schema.js:525]: i feel like these changes came in erroneously from another PR?

Comment (jlipps) [reply packages/appium/test/fixtures/flattened-schema.js:525]: or oh wait this is probably generated from that stuff.

**Review by jlipps — APPROVED** [review]

File: packages/appium/lib/schema/schema.js (hunk 1 above)
Comment (boneskull) [reply packages/appium/lib/schema/schema.js]: I tossed this anyway

File: packages/appium/test/fixtures/flattened-schema.js (hunk 2 above)
Comment (boneskull) [reply packages/appium/test/fixtures/flattened-schema.js:525]: no, neither; this file needed updating manually as per my comment above in the issue

**Comment by boneskull** [comment]
I can't work on this any more today and will pick it up tomorrow.

the root cause has something to do with the recent changes to auto-discovery of extensions.  as I mentioned in slack, I'm unable to reproduce this locally (so it makes fixing it considerably slower).   there's something in the CI environment that is causing fake-driver to be installed twice in a temp dir.  it _seems_ like `appium driver list` does _not_ list `fake`, but upon `appium driver install fake`, it's already installed.  this may have something to do with CI caching, so disabling that is probably the first thing I'd check.  next, running with `_FORCE_LOGS=1` may give further insight into the `npm` commands being issued.  also running a sanity check on `extensions.yaml` and the contents of the temp dir before the code causing the exception runs...

`appium driver list` _should_ cause locally-installed extensions to be auto-detected (which will update `extensions.yaml`), so that's correct, but it's unknown why the driver would be considered locally-installed unless the temp dir's `node_modules` is somehow in an unexpected state.

File: packages/appium/test/fixtures/flattened-schema.js (hunk 2 above)
Comment (jlipps) [reply packages/appium/test/fixtures/flattened-schema.js:525]: 👍

File: packages/appium/test/e2e/plugin.e2e.spec.js [hunk 3]
```diff
@@ -67,13 +71,17 @@ describe('FakePlugin', function () {
...
+        json: true,
+        suppressOutput: true,
       },
       pluginConfig
     );
     if (!_.has(pluginList, 'fake')) {
       await runExtensionCommand(
         {
           pluginCommand: 'install',
+          subcommand: 'plugin',
```
Comment (mykola-mokhnach) [inline packages/appium/test/e2e/plugin.e2e.spec.js:84]: PLUGIN_TYPE?

Comment (boneskull) [reply packages/appium/test/e2e/plugin.e2e.spec.js:84]: good catch thanks

**Comment by boneskull** [comment]
OK, so I've confirmed that `extensions.yaml` is clean when we go to install.  This means that maybe the schema registration is happening twice...

**Comment by boneskull** [comment]
when this is green I'll just squash the whole mess

**Pull request merged**